search-service/src/main/kotlin/com/egm/stellio/search/authorization/SubjectReferentialService.kt

```diff
@@ -161,33 +162,22 @@ class SubjectReferentialService(
             )
             .oneToResult { toInt(it["count"]) }
 
-    suspend fun hasStellioAdminRole(sub: Option<Sub>): Either<APIException, Boolean> =
-        hasStellioRole(sub, GlobalRole.STELLIO_ADMIN)
+    suspend fun hasStellioAdminRole(uuids: List<Sub>): Either<APIException, Boolean> =
+        hasGlobalRoles(uuids, ADMIN_ROLES)
 
-    suspend fun hasStellioRole(sub: Option<Sub>, globalRole: GlobalRole): Either<APIException, Boolean> =
+    suspend fun hasGlobalRoles(uuids: List<Sub>, roles: Set<GlobalRole>): Either<APIException, Boolean> =
```

Comment:
rename it to `hasOneOfGlobalRoles` (otherwise, can't know if it's an and or an or)